internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -183,9 +181,9 @@ func (v *vmConsoleProxy) Cleanup(request *common.Request) ([]common.CleanupResul
 
 func reconcileServiceAccount(serviceAccount core.ServiceAccount) common.ReconcileFunc {
 	return func(request *common.Request) (common.ReconcileResult, error) {
-		serviceAccount.Namespace = getVmConsoleProxyNamespace(request)
+		serviceAccount.Namespace = request.Instance.Namespace
```

Comment:
Deployment of template validator is using `request.Namespace`. What's the difference?

See e.g. https://github.com/kubevirt/ssp-operator/blob/fc1d67c1b3ee193d02cab94149217525975c1f0c/internal/operands/template-validator/reconcile.go#L96

Reply:
IMO, both should return the same value.

1. `request.Instance.Namespace` as a reference to the SSP object, where is deployed so then it makes sense to me to use it.
2. `request.Namespace` seems like a requested namespace where the SSP object should be reconciled, thus I don't think `request.Namespace` should be used in operands but in `ssp_controller` [1].

Tekton operand is using `request.Instance.Namespace` as well [2]:
```
func getTektonTasksNamespace(request *common.Request) string {
	if request.Instance.Spec.TektonTasks != nil && request.Instance.Spec.TektonTasks.Namespace != "" {
		return request.Instance.Spec.TektonTasks.Namespace
	}
	return request.Instance.Namespace
}
```

@akrejcir @ksimon1 WDYT?

[1] https://github.com/kubevirt/ssp-operator/blob/main/controllers/ssp_controller.go#L129-L164
[2] https://github.com/kubevirt/ssp-operator/blob/main/internal/operands/tekton-tasks/reconcile.go#L304-L309

Reply:
Both are the same value, so it does not matter.